src/components/m-header-navigation/js/stroke.js

```diff
@@ -87,8 +87,7 @@ class Stroke extends UiEvents {
     if (has(node, this.options.activeClass)) {
       this._activeNode = node;
 
-      add(node, this.options.activeOpenClass);
-      add(node, this.options.activeMoveClass);
+      add(node, this.options.activeOpenClass, this.options.activeMoveClass);
```

Comment:
@LucaMele 
@TheDadi 
Is it really worth this update?
I checked all other components, only the stroke needs to `add`/`remove` multiple classes at once on the same DOM node.
Valid Reason for me are:
- Pattern-Library consumers won't need to worry if they need to attach multiple classes
- It is more coerce the `Element.classList` API https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
@LucaMele 
@TheDadi 
Knock knock

Reply:
:)